code/WorkInProgress/rewardsLocker.dm

```diff
@@ -283,6 +283,41 @@
 		new /obj/item/pod/paintjob/tronthing(get_turf(activator))
 		return 1
 
+/datum/achievementReward/respirator
+	title = "(Skin) Gas Respirator"
+	desc = "Replaces a gas mask you're wearing with a gas respirator. Use inhand to change style."
+	required_medal = "Old Enemy"
+	once_per_round = FALSE
+	var/sacrifice_path = /obj/item/clothing/mask/gas
+	var/reward_path = /obj/item/clothing/mask/gas/respirator
+
+	rewardActivate(var/mob/activator)
+		if (!istype(activator))
+			return
+
+		if (activator.wear_mask && istype(activator.wear_mask, /obj/item/clothing/mask/gas))
+			var/O = locate(sacrifice_path) in activator.contents
+			var/found = FALSE
+			
+			if (istype(O, sacrifice_path))
```

Comment:
Pretty sure if you're wearing something it will be in your contents so the check and the `found` code aren't necessary.